src/styles/blocks/catalog.scss

```diff
@@ -11,7 +11,7 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
+    transition-duration: var(--animation-duration);
```

Comment:
Please specify to which exactly properties you want to apply the transition. Currently, it applies to all the properties and it's not good for optimization(
https://developer.mozilla.org/en-US/docs/Web/CSS/transition